packages/core/src/App/Containers/RealAccountSignup/personal-details.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len */
```

Comment:
Can we remove this and put it for the line instead?

Reply:
Yes. Actually we can get rid of it now.